src/reference/asciidoc/whats-new.adoc

```diff
@@ -100,6 +100,8 @@ See <<amqp-inbound-channel-adapter>> for more information.
 The `DefaultHttpHeaderMapper.userDefinedHeaderPrefix` property is now an empty string by default instead of `X-`.
 See <<http-header-mapping>> for more information.
 
+Add support for AsyncRestTemplate for outbound channel adapter and gateway. See <<AsyncHttpRequestExecutingMessageHandler>> for more information.
```

Comment:
This sentence should be moved to the `New Components` sections.
And consider to use in the future an Adoc convention when each sentence should be on its own new line.

Plus would be great always keep code mentioning in code style. I mean that `AsyncRestTemplate` should be wrapped with "`" symbol